src/Tizen.NET.MaterialComponents/Components/MList.cs

```diff
@@ -0,0 +1,182 @@
+            {
+                var style = item.Style;
+                var itemStyle = style == MListStyle.OneLine ? Styles.GenListItem.SingleLine : (style == MListStyle.DoubleLine ? Styles.GenListItem.DoubleLine : Styles.GenListItem.TripleLine);
+
+                var defaultClass = new GenItemClass(itemStyle)
+                {
+                    GetTextHandler = (obj, part) =>
+                    {
+                        if (itemStyle != Styles.GenListItem.SingleLine)
+                        {
+                            if (part == Parts.List.SubText)
+                            {
+                                return ((MItem)obj).SubText;
+                            }
+                            else if (part == Parts.List.MetaText)
+                            {
+                                return ((MItem)obj).MetaText;
+                            }
+                        }
+
+                        return ((MItem)obj).Title;
+                    },
+                    GetContentHandler = (obj, part) =>
+                    {
+                        if (part == Parts.List.Icon)
+                        {
+                            if (!string.IsNullOrEmpty(((MItem)obj).Icon))
+                            {
+                                var image = new Image(this)
+                                {
+                                    MinimumWidth = IconSize,
+                                    MinimumHeight = IconSize
+                                };
+                                image.Load(Path.Combine(Applications.Application.Current.DirectoryInfo.Resource, ((MItem)obj).Icon));
+                                return image;
+                            }
+
+                            return null;
+                        }
+
+                        return ((MItem)obj).Control;
+                    }
+                };
+
+                item.GenItem = Append(defaultClass, item);
+
+                item.GenItem.SetPartColor(Parts.Widget.Background, _defaultBackgroundColor);
+                item.GenItem.SetPartColor(Parts.Widget.BackgroundPressed, _defaultActiveBackgroundColor);
+                item.GenItem.SetPartColor(Parts.Widget.BackgroundDisabled, _defaultBackgroundColorForDisabled);
+
+                item.GenItem.SetPartColor(Parts.Widget.Text, _defaultTextColor);
+                item.GenItem.SetPartColor(Parts.Widget.TextPressed, _defaultActiveTextColor);
+            }
+        }
+    }
+
+    public enum MListStyle
+    {
+        OneLine,
+        DoubleLine,
+        TripleLine
+    }
+
+    public class MItem
+    {
+        public MListStyle Style { get; set; }
+        public string Icon { get; set; }
+
+        public string Title { get; set; }
+
+        public string SubText { get; set; }
+
+        public string MetaText { get; set; }
+
+        public EvasObject Control { get; set; }
+
+        public MItem(string title, MListStyle style = MListStyle.OneLine)
```

Comment:
Any reason to provide optional parameter only for style?

Reply:
Done